### DISCUSSION TITLE
how to let the card have colour?

### DISCUSSION THREAD

**Discussion by Pierrelai99** [Budibase/budibase#4496] [Help]
![image](https://user-images.githubusercontent.com/99549872/154010921-d656d07b-803f-4af0-95c2-380340cb4de9.png)
since I add this card, setting there cannot add colour to it, is that got another way to add colour mah?

**Comment by mjashanks** [comment]
Unfortunately the card just doesn't have a colour setting. The Card component could do with some love.

What you can do instead is build up a card yourself, using a Container - then you can put whatever you need inside there too.

The card is really a quick and easy way to get the data on screen, but lacks customisation.

**Reply by Pierrelai99** [reply]
ok thanks ya
